examples/layer2/aws.yml

```
@@ -0,0 +1,56 @@
+---
+# NOTE: this playbook should be run _after_:
+# 1. Running the metal.yml playbook
+# 2. Redeeming the Fabric service token in the Fabric portal
+# 3. Accepting the Direct Connect request in the AWS console
+- name: Equinix Layer 2 example -- AWS resources
+  hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Include the required variables
+      include_vars: "vars/vars.yml"
+
+    - name: create a VPC
+      amazon.aws.ec2_vpc_net:
+        region: "{{ aws_region }}"
+        name: ansible-equinix-layer2-example
+        cidr_block: "{{ aws_network_cidr }}"
```

```suggestion
        name: "{{ vpc_name }}"
```
Make the vpc name a variable.

---

Moved all of the hard-coded names to variables